Makefile

```diff
@@ -46,7 +46,7 @@ tox: venv
 	tox
 
 flake:
-	flake8 hy tests --ignore=E121,E123,E126,E226,E24,E704,W503,E305
+	flake8 hy tests --ignore=E121,E123,E126,E226,E24,E704,W503,E302,E305,E701
```

Comment:
That would be a huge swarm of `noqa` comments, but besides, I don't think these style rules are useful. I've been bitten by the newline one, in particular, dozens of times in writing Hy patches, and in each case the organization of the code only became less clear when I added more space, because I was using the lack of space to visually group related constructs. Likewise I don't see the value in banning `with ...: ...` or `if ...: ...` or whatever. I agree that in *most* cases you want the newline after the colon, but sometimes you don't.